k8s/agent/Dockerfile

```diff
@@ -0,0 +1,52 @@
+FROM ubuntu:18.04
+
+ENV DEBIAN_FRONTEND="noninteractive"
+
+RUN apt-get update -y && \
+    apt-get upgrade -y && \
+    apt-get install -y software-properties-common build-essential git  \
+                        jq python3 python3-dev python3-pip python3-setuptools \
+                        libpython3-dev locales apt-transport-https file wget curl \
+                        sudo iputils-ping openssh-client libpq-dev sshpass && \
+    apt-get clean && apt-get autoclean && \
+    rm -rf \
+        /var/lib/apt/lists/* \
+        /tmp/* \
+        /var/tmp/* \
+        /usr/share/man \
+        /usr/share/doc \
+        /usr/share/doc-base
+
+COPY k8s/agent/default_locale /etc/default/locale
+RUN locale-gen en_US.UTF-8 && update-locale LANG=en_US.UTF-8 && \
+    chmod 0755 /etc/default/locale
+
+# Install Kops
+ENV KOPS_VERSION=1.10.0
```

Comment:
Verify the version of kops and helm: https://github.com/legion-platform/legion/blob/develop/k8s/ansible/Dockerfile#L32